### PR TITLE
Fix `dim-bots` with multiple PR labels

### LINKED ISSUE #4354: Dim bots in user pull request view
### Description

I would like to have the Pull Request reviews for dependencies (dependabot, renovate) dimmed in the user Pull Request view (https://github.com/pulls) and repository pull request view (https://github.com/{org}/{repo}/pulls) when it has multiple label and not only the `bot` label.

### PR DESCRIPTION
**Pull request by agaudreault-jive**
Closes #4354
Fixes a regression from #3921 causing the PRs with more than just the `bot` label not to be dimmed anymore.

## Test URLs

1. Create a pull request with a `bot` label and any other one.
2. Browse to https://github.com/pulls

## Screenshot

|     | normal | hover |
| --- | ------ | ----- |
| Before | image | N/A |
| After | image | image |

### CODE REVIEW

**Comment by yakov116** [comment]
Thanks for the PR
Please fill out the entire template

**Comment by agaudreault-jive** [comment]
@yakov116 I am not very sure about the selector syntax, but while I was testing, I found out that 
`'.labels [href$="label%3Abot"]'` selector will not work if the PR has more than one label.

Do you know the syntax for `labels.contains('bot')` so I could fix that at the same time ?

File: source/features/dim-bots.tsx [hunk 1]
```diff
@@ -4,30 +4,37 @@ import * as pageDetect from 'github-url-detection';
...
+		bot.closest('.commit, .Box-row')!.classList.add('rgh-dim-bot');
+	}
 }
 
 void features.add(__filebasename, {
 	include: [
 		pageDetect.isCommitList,
-		pageDetect.isConversationList
+		pageDetect.isConversationList,
+		pageDetect.isPRList
```
Comment (fregante) [inline source/features/dim-bots.tsx:37]: PRList is also already a conversation list if I remember correctly

Comment (agaudreault-jive) [reply source/features/dim-bots.tsx:37]: Doesn't seem so, see https://github.com/fregante/github-url-detection/blob/master/index.ts#L205

File: source/features/dim-bots.tsx [hunk 2]
```diff
@@ -4,24 +4,30 @@ import * as pageDetect from 'github-url-detection';
...
 	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
+	'.commit-author[href$="scala-steward"]'
+].join();
 
+const prSelector = [
 	/* Issues/PRs */
 	'.opened-by [href*="author%3Aapp%2F"]',
 	'.labels [href$="label%3Abot"]'
 ].join();
```
Comment (yakov116) [inline source/features/dim-bots.tsx:18]:
```suggestion
];
```

`select` takes care or the join

**Comment by yakov116** [comment]
I found it on https://github.com/pulls?page=2&q=archived%3Afalse+commenter%3Aagaudreault-jive+sort%3Aupdated-desc

**Review by yakov116 — APPROVED** [review]
LGTM

**Comment by yakov116** [comment]
Thanks @agaudreault-jive

**Pull request merged**